resources/static/css/app.css

```diff
@@ -713,6 +717,13 @@ div.spinner {
   opacity: 1;
 }
 
+.question {
```

Comment:
did you ask @manuelwedler why he deleted it in the first place? Was it just a mistake?

Reply:
Well the sass compiler deleted it, because Sohan didn't add it to `.scss` file

Reply:
@Dominik1999 I was ignorant about working with Sass, we only use the sass file for css changes, which compiles to the `app.css` files. So we never touch the `app.css` file the sass compiler should do it for us.

Reply:
perfect, thanks, then merge